proto/paladin.proto

```diff
@@ -47,13 +47,29 @@ message PaladinTalents {
 
 message RetributionPaladin {
     message Rotation {
-		bool consecration = 1;
+		enum ConsecrateRank {
+			None = 0;
+			Rank1 = 1;
+			Rank4 = 2;
+			Rank6 = 3;
+		}
+		ConsecrateRank consecrateRank = 1;
+		bool exorcism = 2;
```

Comment:
Very nitpicky, but could you name this 'use_exorcism' just to be consistent with similar options in other sims?